### PR TITLE
Add support for Intel Macs, standalone GPUs, add requirements.txt

### PR DESCRIPTION
**Pull request by elrid**

### CODE REVIEW

**Comment by madebyollin** [comment]
Thanks! I just removed `Float16` in an alternate way (via cast inside the MPSGraph). Does it work on Intel now? If so, we can simplify this PR a bit (no more Accelerate) and merge it.

**Comment by elrid** [comment]
Yep, everything now works on intel macs:)

File: maple-diffusion/MapleDiffusion.swift [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import MetalPerformanceShadersGraph
 import Foundation
+import Accelerate
```
Comment (madebyollin) [inline maple-diffusion/MapleDiffusion.swift]: This import is no longer needed, right? Other than that LGTM

Comment (elrid) [reply maple-diffusion/MapleDiffusion.swift]: Oh, yeah, sorry, missed it:)

**Comment by madebyollin** [comment]
Merged (and I updated the README section regarding Intel support). Thanks again!

**Pull request merged**